package.json

```diff
@@ -14,7 +14,6 @@
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
     "babel-eslint": "^10.0.3",
-    "babel-jest": "^24.9.0",
```

Comment:
If we remove this, how does Jest know to work with Babel? Is it just built in?

Reply:
Apparently this is now baked into jest core: https://github.com/babel/babel-jest
(This is my 1 thing to learn today, as I just discovered it)

Reply:
keep it in - while it's included it only works due to npm/yarn hoisting

Reply:
Yup, added it back in. Whilst the functionality (auto-babel detection) is now part of the core Jest logic, it turns out you still need this package; and this package is also part of Jest core packages.